src/mempool_options.h

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2022 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+#ifndef BITCOIN_MEMPOOL_OPTIONS_H
+#define BITCOIN_MEMPOOL_OPTIONS_H
+
+#include <mempool_limits.h>
+
+#include <chrono>
+#include <cstddef>
+#include <cstdint>
+
+class CBlockPolicyEstimator;
+
+/** Default for -maxmempool, maximum megabytes of mempool memory usage */
+static constexpr unsigned int DEFAULT_MAX_MEMPOOL_SIZE_MB{300};
+/** Default for -mempoolexpiry, expiration time for mempool transactions in hours */
+static constexpr unsigned int DEFAULT_MEMPOOL_EXPIRY_HOURS{336};
+
+/**
+ * Options struct containing options for constructing a CTxMemPool. Default
+ * constructor populates the struct with sane default values which can be
+ * modified.
+ *
+ * Most of the time, this struct should be referenced as CTxMemPool::Options.
+ */
+struct MemPoolOptions {
+    /* Used to estimate appropriate transaction fees. */
+    CBlockPolicyEstimator* const estimator = nullptr;
```

Comment:
For all commits: Any reason to not use `{}` over `=` in this struct for C++11 member inits?

Reply:
Nope! Is `{}` preferred?

Reply:
Yes, for new code it is preferred style. `{}` has narrowing warnings enabled, as opposed to `=` or `()`, which do not.

Reply:
Closing as resolved